dask/array/reductions.py

```diff
@@ -313,7 +313,12 @@ def partial_reduce(
             meta = func(reduced_meta, computing_meta=True)
         # no meta keyword argument exists for func, and it isn't required
         except TypeError:
-            meta = func(reduced_meta)
+            try:
+                meta = func(reduced_meta)
+            except ValueError as e:
+                # min/max functions have no identity, don't apply function to meta
+                if "zero-size array to reduction operation" in str(e):
```

Comment:
Is it worth considering checking the name of the function and special casing min/max. This isn't great but we can check the name with something like:
> func.funcs[0].func.__name__

Reply:
I think it's not necessary, this is already the best we can do in that case, so even if there's another function that fails the same way, this will probably be our best shot at resolving that issue.